### PR TITLE
Omit Milestone model

### PR DESCRIPTION
**Pull request by priyanshisharma**
@anshsrtv if you opened this first, then go to #1 and then come back here 😅 

As described in #1 I'm not sure whether making a new model for a few boolean values complicates stuff or makes it easy , so I just made 2 PRs 😂😂, Take the liberty to choose if you find sense in any of the two and close the other one 😄

### CODE REVIEW

File: profiles/models.py [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
-
 class Leaderboard(models.Model):
     username = models.ForeignKey(User, on_delete=models.CASCADE)
     points = models.IntegerField()
-    milestones = models.ForeignKey(Milestone, on_delete=models.CASCADE)
+    good_first_issue = models.BooleanField(default=False) #Shall be true on completion of Good first issue
+    medium_question_1 = models.BooleanField(default=False) #Shall be true on completion of first medium issue
+    medium_question_2 = models.BooleanField(default=False) #Shall be true on completion of first medium issue
+    '''
+    We can identify labels for various situations as follows
```
Comment (anshsrtv) [inline profiles/models.py:11]: labels here means?? Sorry if I sound dumb... :joy:

Comment (priyanshisharma) [reply profiles/models.py:11]: No you don't sound dumb, and by labels I meant `good_start`, `terrific_start`, `almost_there` and `mission_accomplished` 😅

Comment (anshsrtv) [reply profiles/models.py:11]: in the views or something... when we send it to the frontend??

Comment (priyanshisharma) [reply profiles/models.py:11]: Yeah

**Pull request merged**